packages/phrases/src/locales/ja/translation/admin-console/api-resources.ts

```diff
@@ -13,7 +13,6 @@ const api_resources = {
   default_api_label:
     'テナントごとにデフォルトのAPIを0または1つだけ設定できます。デフォルトのAPIが指定されている場合、認証リクエストでリソースパラメータを省略できます。その後のトークン交換は、デフォルトのAPIを対象として行われます。それにより、JWTが発行されます。<a>詳細を見る</a>',
   api_resource_created: 'APIリソース{{name}}が正常に作成されました',
-  /** UNTRANSLATED */
   invalid_resource_indicator_format: 'API indicator must be a valid absolute URI.',
```

Comment:
ditto. should have more like this

Reply:
it's not practical to check files one by one though. i'm good with some missing phrases

Reply:
Discussed offline and we are OK to leave it as-is